plugins/modules/aws_s3_cors.py

```diff
@@ -143,6 +145,14 @@ def destroy_bucket_cors(connection, module):
     module.exit_json(changed=changed)
 
 
+def fix_list_dict(camel_rules):
+    fixed_rules = []
+    for camel_rule in camel_rules:
+        fixed_rules.append(ast.literal_eval(camel_rule))
```

Comment:
Does not work for me, because the key value pair in rules is maleformed.

```
ValueError: malformed node or string: {'allowed_headers': 'Authorization'}
```

furthermore I don't know if a saftey evaluation is necessary here, because `rules` is already verified by `AnsibleAWSModule()`.